### PR TITLE
Update Index.md

### PR DESCRIPTION
**Pull request by khushi-purwar**
Added Blog Application Project and FAQ Application Project and Typing Speed Test Website  Project in the index.md

### CODE REVIEW

File: Index.md [hunk 1]
```diff
@@ -44,15 +45,15 @@
...
 | [BMI Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/bmi-calculator)                           | A basic website which can be used to calculate body mass index |
 | [Glassmorphism Animation Form](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Glassmorphism-Form)      |A basic login page with Glassmorphism Animations |       
-| [Battery Percentage Indicator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Battery%20Indicator)      |This is a battery indicator app which is used to know battery percentage of the system. |
-| [ConnectFour](https://github.com/Shauryaditya/Web-dev-mini-projects/tree/main/ConnectFour)                                  |  This is a simple game which can be played between two players.The player who connects the four bubbles first wins the game . The bubbles could be connected horizontally , vertically or diagonally.The players have to be together since multiplayer feature is not available .|
```
Comment (chicken-biryani) [inline Index.md:58]: You've removed connect fouur. Just re-edit and this is good to go.

Comment (khushi-purwar) [reply Index.md:58]: > You've removed connect fouur. Just re-edit and this is good to go.

Done!

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Hey @khushi-purwar, please resolve conflicts and add the details of #195 in the index.md file.

File: Index.md [hunk 2]
```diff
@@ -37,28 +39,25 @@
...
 | [News Webapp](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/flask_news_project)                            | A web app for searching news made using flask                                                                                                                                                                                        |
-| [Score Keeper](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Score-Keeper)                                  |A web Page where user can keep a count of their scores and will tell which player won.|
-| [Image Filter App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Image-Filter-App)                                                                                                            |An Awesome Image Filter App written in Html,Css,Javascript and CamanJS.|
+| Score Keeper                                                                                                                  |A web Page where user can keep a count of their scores and will tell which player won.|
```
Comment (harshita2216) [inline Index.md:54]: Removed link?

File: Index.md [hunk 3]
```diff
@@ -37,28 +39,25 @@
...
+| [Image Filter App]()                                                                                                            |An Awesome Image Filter App written in Html,Css,Javascript and CamanJS.|
 | [Form Validation](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/form-validation)                         |A basic sign up page with all the validations in javascript.   |     
 | [Age Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/age-calculator)                           | A website which can be used to calculate age of a person in days,months and years . |
 | [BMI Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/bmi-calculator)                           | A basic website which can be used to calculate body mass index |
-| [Day Of The Week App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Day-Of-The-Week-App)                       | A basic app that displays current day of the week with quotes . |
```
Comment (harshita2216) [inline Index.md:56]: Removed?

File: Index.md [hunk 4]
```diff
@@ -37,28 +39,25 @@
...
-| [Battery Percentage Indicator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Battery%20Indicator)      |This is a battery indicator app which is used to know battery percentage of the system. |
+| [Battery Percentage Indicator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Battery%20Indicator)      |This is a battery indicator app which is used to know battery percentage of the system. | 
 | [ConnectFour](https://github.com/Shauryaditya/Web-dev-mini-projects/tree/main/ConnectFour)                                  |  This is a simple game which can be played between two players.The player who connects the four bubbles first wins the game . The bubbles could be connected horizontally , vertically or diagonally.The players have to be together since multiplayer feature is not available .|
-| [Expand Button](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Expand-Button) |It is a basic website with an expanding button on it. As we click on the button it expands and shrinks after 5 seconds. |
```
Comment (harshita2216) [inline Index.md:60]: Removed?

**Review by harshita2216 — CHANGES_REQUESTED** [review]

**Comment by khushi-purwar** [comment]
> Hey @khushi-purwar, please resolve conflicts and add the details of #195 in the index.md file.

Resolved the conflicts and added #195.

**Comment by harshita2216** [comment]
> > Hey @khushi-purwar, please resolve conflicts and add the details of #195 in the index.md file.
> 
> Resolved the conflicts and added #195.

Check review please! @khushi-purwar

File: Index.md (hunk 4 above)
Comment (khushi-purwar) [reply Index.md:60]: I am just adding my project link, don't know why other projects are changing. Sorry for my mistake.

**Comment by harshita2216** [comment]
Why are you removing the entire code and copying everything again? 
You are only supposed to add your project details. Keep the other things untouched. 
Update your fork and take a fresh pull and this time only add your details. @khushi-purwar

**Pull request closed without merging**